web/modules/custom/bnf/bnf.info.yml

```diff
@@ -2,6 +2,4 @@ name: 'BNF Common'
 type: module
 description: 'Common code between BNF (Bibliotekernes Nationale Formidling) client and server modules'
 package: 'DPL - BNF'
-# It doesn't make sense to enable this module on its own.
```

Comment:
Then put `$settings['extension_discovery_scan_tests'] = TRUE` in your settings file.